addons/binding/org.openhab.binding.rme/ESH-INF/thing/rme.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rme"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="manager">
+		<label>RME Rain Manager</label>
+		<description>Thing for RME Rain Manager</description>
+
+		<channels>
+			<channel id="waterlevel" typeId="waterlevel" />
+			<channel id="mode" typeId="mode" />
+			<channel id="source" typeId="source" />
+			<channel id="exitpump" typeId="statusflag" />
+			<channel id="entrypump" typeId="statusflag" />
```

Comment:
You can by now (and should!) provide individual labels/descriptions for each channel, so that they do not all use the same label "Status Flag" from the channel type.